dataclients/kubernetes/testdata/ingressV1/ingress-data/ing-with-duplicate-host.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: foo
+  name: qux
+spec:
+  defaultBackend:
+    service:
+      name: bar
+      port:
+        number: 1234
+  rules:
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/"
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: bar
+            port:
+              name: baz
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/"
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: bar
+            port:
+              name: baz
```

Comment:
Lets also have a test case for the same host and the same path but different services